downstream/assemblies/platform/assembly-aap-platform-components.adoc

```diff
@@ -8,7 +8,17 @@ ifdef::context[:parent-context: {context}]
 
 
 [role="_abstract"]
-{PlatformNameShort} is a modular platform composed of separate components that can be connected together to meet your deployment needs. {PlatformNameShort} deployments start with {ControllerName} which is the enterprise framework for controlling, securing, and managing Ansible automation with a user interface (UI) and RESTful application programming interface (API). Then, you can add to your deployment any combination of the following automation platform components:
+{PlatformNameShort} is a modular platform composed of separate components that are connected together to meet your deployment needs. {PlatformNameShort} deployments start with {ControllerName}, which is the enterprise framework for controlling, securing, and managing Ansible automation with a user interface (UI) and RESTful application programming interface (API). Deploy each of the following components so that all features and capabilities are available for use without the need to take further action:
```

Comment:
How about this

{PlatformNameShort} is composed of services that are connected together to meet your automation needs. These services provide the ability to store, make decisions for, and execute automation. All of these functions are available through with a user interface (UI) and RESTful application programming interface (API). Deploy each of the following components so that all features and capabilities are available for use without the need to take further action:

Reply:
Feel free to modify my writeup as you like. Essentially we don't want to give the thought that its optional to deploy, nor really lean into the service names themselves except as a thing to be deployed. The function is used at the platform level, not at the service level.

Reply:
That rewrite looks good to me, @thaumos! I have made that change. Thank you for the contextual information!